ts/WoltLabSuite/Core/Acp/Ui/User/Action/AbstractUserAction.ts

```diff
@@ -0,0 +1,17 @@
+export abstract class AbstractUserAction {
```

Comment:
Missing class comment

Reply:
This is not yet resolved, because the class itself is not commented.